src/Module.Server/Modes/Dtv/CrpgDtvServer.cs

```diff
@@ -174,6 +174,11 @@ private void StartNextRound()
         _currentWave = -1;
         SpawningBehavior.RequestSpawnSessionForRoundStart(firstRound: _currentRound == 0);
         SendDataToPeers(new CrpgDtvRoundStartMessage { Round = _currentRound });
+        foreach (Agent mount in Mission.MountsWithoutRiders) // force mounts to flee
+        {
+            mount.CommonAIComponent?.Panic();
```

Comment:
why are you declaring CommonAiComponent  as nullable?

Reply:
I took inspiration from the `MakeEnemiesFleeCheat` method within the `Mission` class which performs a null check on the CommonAIComponent. In hindsight I guess this is because it is possibly iterating through non-AI Agents. 

In this case I don't see an occasion where MountsWithoutRiders would have Agents without a CommonAIComponent so I suppose making this nullable isn't necessary! I will test this ingame first.